README.md

```diff
@@ -1 +1,159 @@
-# builder-react-component
+[![Travis Status][trav_img]][trav_site]
+
+Builder Archetype: React Component
+==================================
+
+A React component archetype for [builder][].
+
+## Project Structure
+
+This archetype assumes an architecture as follows:
+
+```
+demo/
+  app.jsx
+  index.html
+src
```

Comment:
should this assume a `components` directory as well? (same for `test/client/spec`)